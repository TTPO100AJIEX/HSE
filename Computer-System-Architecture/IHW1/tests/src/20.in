10
-3 34 7 20 34 26 -4 -19 0 -9