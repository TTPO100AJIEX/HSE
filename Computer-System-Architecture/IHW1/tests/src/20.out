-3 34 7 20 34 -19 0 